Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Use the official Go image as the base for building
+FROM golang:1.23 as builder
+
+# GitHub tag
+ARG RELEASE_VERSION
+
+# Set the working directory
+WORKDIR /app
+
+# Download the source code from GitHub Releases
+RUN curl -L https://github.com/pingidentity/pingcli/archive/refs/tags/v${RELEASE_VERSION}.tar.gz | tar xz --strip-components=1
+
+# Build the pingcli binary (ensure static build)
+RUN CGO_ENABLED=0 go mod tidy && CGO_ENABLED=0 go build -o /pingcli
```

Comment:
Curling source and building pingcli is unneeded: https://goreleaser.com/customization/docker/#the-docker-build-context

Also, this would be why the version and commit uuid are not being populated by goreleaser.